packages/engine/src/assets/compression/ModelTransformFunctions.ts

```diff
@@ -343,9 +343,24 @@ function hashBuffer(buffer: Uint8Array): string {
   return hash.digest('hex')
 }
 
+enum Status {
+  Initializing,
+  ProcessingTexture,
+  Finalizing,
+  WritingFiles,
+  Complete
+}
+
+export { Status as ModelTransformStatus }
+
 export async function transformModel(
   args: ModelTransformParameters,
-  onMetadata: (key: string, data: any) => void = (key, data) => {}
+  onMetadata: (key: string, data: any) => void = (key, data) => {},
+  onProgress: (progress: number, status: Status, numerator?: number, denominator?: number) => void = (
+    numerator,
+    denominator,
+    status
```

Comment:
```suggestion
    _numerator,
    _denominator,
    _status
```

we can use `_` underscore to denote unused variable in this case

Reply:
These are in a function declaration that's a default value for a function parameter, so I'll just delete the default value and make the function parameter optional.

Believe it or not, I think the formatter wrote these out. I was using `() => {}` as the default value and I guess the formatter didn't like my arg erasure. It's a funny old world